### ISSUE TITLE
Remove content shifting when Modal is opened.

### ISSUE THREAD

**Issue by b-novikov-ipersonality** [chakra-ui/chakra-ui#5705]
### Description

When modal is open, scroll bar is removed, shifting content. I suggest to tweak styles to change this behavior 

### Problem Statement/Justification

For me as a user it's unpleasant to see content of a webpage shift. I consider it bad UX because it's kinda pushes away from resource. 

### Proposed Solution or API

Solution found on stackoverflow:

```css
html {
    margin-right: calc(100% - 100vw);
}
body {
    margin-right: calc(100vw - 100%);
}
```
works for me well. But maybe there's a better solution.

### Alternatives

Manually style html and body as per proposed solution.

### Additional Information

https://stackoverflow.com/a/47276144

**Comment by TylerAPfledderer** [comment]
Hey @b-novikov-ipersonality!

Chakra implements an offset along with other styles and functionality for the scroll-bar via the [react-remove-scrollbar](https://github.com/theKashey/react-remove-scroll-bar) package.

You can also see it applied in the [modal source file](https://github.com/chakra-ui/chakra-ui/blob/main/packages/modal/src/modal.tsx) line 317 and the styling is being applied in the doc page.

I wonder if there is something overriding it? 🤔

Is it breaking on your end when you use the examples on the doc page?

**Comment by b-novikov-ipersonality** [comment]
@TylerAPfledderer examples in chakra docs work fine. Unlike in chakra modal example, on my end shifting happens because the scroll bar is removed. 

I narrowed the problem down to global styles in my custom theme - settings html, body and root element width to 100% causes content to shift when modal is opened. I removed width property from global styles, now content don't shift, except header which has position fixed and width 100%, but I will figure it out. 

Thank you for help, Tyler.
